### PR TITLE
Deploy parser

### PR DESCRIPTION
**Pull request by DevAseel**
**Summary of change:**
- Added a new micro-service, deploy parser, which takes a fetched deploy and parse in a specific format, so it can be saved in the datasource. 

**List of tasks:** 
- [x] Define types.
- [x] Define interfaces.
- [x] Define class.
- [x] Create fixture. 
- [x] Create unit test.
- [x] Generate the docs. 
- [x] Update the documentation.

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/deploy-parser`
```bash
git checkout feature/deploy-parser
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm deployFetcher test has passed.
- Run the docs using live server and make sure it have been updated.

### CODE REVIEW

**Review by specialistvlad — APPROVED** [review]

File: src/services/parsers/DeployParser.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        dependencies: deploy.fields.deploy.header.dependencies,
+        chainName: deploy.fields.deploy.header.chain_name,
+        parsedPaymentAmount: deploy.fields.deploy.payment.ModuleBytes.args[0][1].parsed,
+        contractHash: deploy.fields.deploy.session.StoredContractByHash.hash,
+        entryPoint: deploy.fields.deploy.session.StoredContractByHash.entry_point,
+        blockHash: deploy.fields.execution_results[0].block_hash,
+        blockNumber: deploy.blockNumber,
+        cost: deploy.fields.execution_results[0].result.Success.cost,
+        payment: deploy.fields.deploy.payment,
+        session: deploy.fields.deploy.session,
```
[the remark this answers is not in the document]
Comment (DevAseel) [reply src/services/parsers/DeployParser.ts:42]: 10-4!

**Comment by specialistvlad** [comment]
@DevAseel could you resolve conflicts?

**Comment by DevAseel** [comment]
@specialistvlad this one should be closed without merging, all changes are included in #[47](https://github.com/fluidefi/fluidefi-caspernet-aggregator-tools/pull/47).

**Comment by specialistvlad** [comment]
Closing this too

**Pull request closed without merging**